File: src/main/java/me/nepnep/nepbot/message/command/commands/Ping.java
package me.nepnep.nepbot.message.command.commands;

import me.nepnep.nepbot.message.command.Category;
import me.nepnep.nepbot.message.command.ICommand;
import net.dv8tion.jda.api.Permission;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

import java.util.List;

public class Ping implements ICommand {
    @Override
    public void execute(List<String> args, GuildMessageReceivedEvent event) {
        event.getChannel().sendMessageFormat("Pong: %d%s", event.getJDA().getGatewayPing(), "ms").queue();
    }
    @Override
    public String getInvoke() {
        return "ping";
    }
    @Override
    public Permission requiredPermission() {
        return null;
    }
    @Override
    public Category getCategory() {
        return Category.GENERAL;
    }
    @Override
    public String getDescription() {
        return "Pong!";
    }
}

File: src/main/kotlin/me/nepnep/nepbot/message/command/commands/Ping.kt
package me.nepnep.nepbot.message.command.commands

import me.nepnep.nepbot.message.command.Category
import me.nepnep.nepbot.message.command.AbstractCommand
import net.dv8tion.jda.api.entities.channel.middleman.GuildMessageChannel
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class Ping : AbstractCommand(
    "ping",
    Category.GENERAL,
    "Pong!"
) {
    override suspend fun execute(args: List<String>, event: MessageReceivedEvent, channel: GuildMessageChannel) {
        channel.sendMessage("${event.jda.gatewayPing}ms").queue()
    }
}